config/dev.exs

```diff
@@ -155,3 +155,9 @@ config :game_client, dev_routes: true
 ##############################
 # App configuration: gateway #
 ##############################
+
+##################################
+# App configuration: bot_manager #
+##################################
+
+config :bot_manager, :end_point, port: [port: 5000]
```

Comment:
I failed to run the app initially because port 5000 is occupied in mac by the control center unless disabled: https://nono.ma/port-5000-used-by-control-center-in-macos-controlce. To avoid anyone else from running into this issue, I think we could change this port

Reply:
Yeah i was thinking the same, also toxi proxy is using this port so a change was needed. Moved to port 4003